doc/whats_new.rst

```diff
@@ -72,9 +72,14 @@ Bug
 - Fix bug where :func:`~hnn_core.network.pick_connection` failed when searching
   for connections with a list of cell types, by `Nick Tolley`_ in :gh:`559`
 
+
```

Comment:
Nitpick: avoid adding unnecessary blank lines.

Reply:
I'll remember this moving forward.